liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy

```diff
@@ -0,0 +1,134 @@
+package com.liferay.blade.tests
+
+import org.gradle.testkit.runner.GradleRunner
+import static org.gradle.testkit.runner.TaskOutcome.*
+import spock.lang.Specification
+import de.undercouch.gradle.tasks.download.Download;
+import java.util.jar.JarInputStream;
+import java.io.FileNotFoundException;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Request.Builder;
+
+class BladeTests extends Specification {
+	public void checkManifest(arg1='', arg2='', arg3='') {
+		def jarInputStream = new JarInputStream(new FileInputStream("${arg1}"))
```

Comment:
Instead of JarInputStream you can just use "new aQute.bnd.osgi.Jar" and call getmanifest().getMainAttributes()